
# HG changeset patch
# User Hans de Goede <hdegoede@redhat.com>
# Date 1264588206 -3600
# Node ID e6c311469ee1929bd422b90fa422fab0c12077eb
# Parent  7d9ba85f62eae4ae2b25c5068cbb2c6ed2d20a10
libv4l: update upsidedown device list

From: Hans de Goede <hdegoede@redhat.com>

libv4l: update upsidedown device list

Priority: normal

Signed-off-by: Hans de Goede <hdegoede@redhat.com>

--- a/v4l2-apps/libv4l/libv4lconvert/control/libv4lcontrol.c	Wed Jan 27 09:33:58 2010 +0100
+++ b/v4l2-apps/libv4l/libv4lconvert/control/libv4lcontrol.c	Wed Jan 27 11:30:06 2010 +0100
@@ -157,7 +157,7 @@
     V4LCONTROL_HFLIPPED|V4LCONTROL_VFLIPPED },
   { 0x04f2, 0xb071, 0, "PEGATRON CORP.", NULL,
     V4LCONTROL_HFLIPPED|V4LCONTROL_VFLIPPED, 0,
-    "Philco", "S.rie PHN10050" },
+    "Philco", "S\202rie PHN10050" },
   { 0x04f2, 0xb072, 0, "ASUSTeK Computer Inc.        ", "K50IJ     ",
     V4LCONTROL_HFLIPPED|V4LCONTROL_VFLIPPED },
   { 0x04f2, 0xb106, 0, "ASUSTeK Computer Inc.        ", "N50Vc      ",

